sdk/core/System.Net.ClientModel/tests/client/MapsClient/MapsClient.cs

```diff
@@ -64,11 +61,11 @@ public virtual Result GetCountryCode(string ipAddress, RequestOptions options =
         if (ipAddress is null) throw new ArgumentNullException(nameof(ipAddress));
 
         options ??= new RequestOptions();
```

Comment:
Note that we don't want to just new RequestOptions now, because that will create a new pipeline and is super inefficient.  We'll want the client to store the PipelineOptions so any new RequestOptions can use the cached pipeline.